cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -483,6 +484,16 @@ public CoachMarkBuilder setDismissOnAnchorDetach(boolean shouldDismissOnAnchorDe
             return this;
         }
 
+        /**
+         * Set the coach mark's text color.
+         *
+         * @param textColor new text color
+         */
+        public CoachMarkBuilder setTextColor(@ColorInt int textColor) {
+            ((TextView) this.content).setTextColor(textColor);
```

Comment:
Is this content view always a `TextView`? 

If that's the case, why not having it as a `TextView` directly to avoid the casting here? (Probably something out of scope of this PR)

If that's not the case, shouldn't we make this API safer and check if content is a `TextView` before setting the text color? I mean, I think it's ok to crash here if someone is misusing the API, but we could do something like the next:

```java
if (this.content instanceOf TextView) {
   ((TextView)this.content).setTextColor(textColor);
} else {
    throw new IllegalStateException("Can't set a text color in a CoachMark whose content is not a TextView");
}

Reply:
Good point! It's only a text view when you use the Builder constructor that takes a string and there are two more constructors.

I've changed this and I've added tests to check we can't set the text color on a non-TextView. I've also added simple tests to check we can show non-TextView coach marks, even though we haven't really added that behaviour here. I wouldn't have made this mistake if those tests had existed.

Could you please have another look @vjgarciag96 @KeniF ?

Reply:
Thanks for the changes! LGTM 👏🏻